test/integration/ink_editor_spec.mjs

```diff
@@ -1294,3 +1294,45 @@ describe("Should switch from an editor and mode to others by double clicking", (
     );
   });
 });
+
+describe("Ink must update its color", () => {
+  let pages;
+
+  beforeEach(async () => {
+    pages = await loadAndWait("empty.pdf", ".annotationEditorLayer");
+  });
+
+  afterEach(async () => {
+    await closePages(pages);
+  });
+
+  it("must check that the stroke color is the one choose from the color picker", async () => {
+    await Promise.all(
+      pages.map(async ([_browserName, page]) => {
```

Comment:
In other places if the parameter is not used we only use an underscore to indicate that; see https://github.com/mozilla/pdf.js/blob/e853a8f41a9ba57b1a556c5ac4bdd39f95ddf8e2/test/integration/signature_editor_spec.mjs#L55, so let's do that there too for consistency: `async ([_, page])`